includes/loadTrack.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
blank line at top . Also this file should be in the JS folder

Reply:
🎉